flytekit/common/tasks/task.py

```diff
@@ -189,14 +190,7 @@ def validate(self):
 
     @_exception_scopes.system_entry_point
     def add_inputs(self, inputs):
-        """
-        Adds the inputs to this task.  This can be called multiple times, but it will fail if an input with a given
-        name is added more than once, a name collides with an output, or if the name doesn't exist as an arg name in
-        the wrapped function.
-        :param dict[Text, flytekit.models.interface.Variable] inputs: names and variables
-        """
-        self._validate_inputs(inputs)
-        self.interface.inputs.update(inputs)
+        raise _user_exceptions.FlyteUserException("You can not add inputs to this task")
```

Comment:
Why is this an SDKRunnableTask-only behavior?

Reply:
I'll let @wild-endeavor chime in on this one

Reply:
@matthewphsmith ?